src/components/container/SurveyTaskList/SurveyTaskList.tsx

```diff
@@ -15,13 +17,19 @@ export interface SurveyTaskListProps {
 	previewState?: SurveyTaskListListPreviewState
```

Comment:
I realize previewState is not new, but it would be helpful if there were a hint somewhere that it is CompleteTasks/IncompleteTasks. Someone could always read the code to find out, but would be much nicer to spare that 5 minutes

Reply:
That's what typescript and autocomplete is for?  I'm not sure how you're suggesting I make it easier?